source/dub/commandline.d

```diff
@@ -960,12 +960,15 @@ class InitCommand : Command {
 				addDependency(dub, p, depspec);
 			}
 		}
-
+		auto pkg = splitPackageName(m_templateType);
+		string[string]depVers;
+		if(pkg.version_ !is null)
+			depVers[pkg.name] = pkg.version_;
 		if (!["vibe.d", "deimos", "minimal"].canFind(m_templateType))
 		{
-			free_args ~= m_templateType;
+			free_args ~= pkg.name;
 		}
```

Comment:
I think this new code fits better inside the createEmptyPackage method, not needing to change anything in the InitCommand and also avoiding to move the depVers logic from `createEmptyPackage` into here.

The if block here should also be moved into there imo, modifying `free_args` because we want to pass in the custom template type as dependency seems more like a hack. The dependency version determination logic is done inside `createEmptyPackage`, so I think it's better to determine the init template dependency version there anyway.